prometheus.yml

```diff
@@ -64,6 +64,8 @@ scrape_configs:
         target_label: __metrics_path__
         replacement: /api/v1/nodes/${1}/proxy/metrics
         action: replace
+      - action: labeldrop
+        regex: (beta_kubernetes_io_arch|beta_kubernetes_io_os|kubernetes_io_arch|kubernetes_io_os|cloud_google_com_gke_max_pods_per_node|cloud_google_com_gke_os_distribution|cloud_google_com_gke_logging_variant|cloud_google_com_gke_container_runtime|cloud_google_com_private_node|cloud_google_com_gke_boot_disk|cloud_google_com_gke_cpu_scaling_level|cloud_google_com_gke_provisioning)
```

Comment:
```regex
((beta_)?kubernetes_io_(arch|os)|cloud_google_com_gke_(max_pods_per_node|os_distribution|logging_variant|container_runtime|boot_disk|cpu_scaling_level|provisioning)|cloud_google_com_private_node)
```

Reply:
Your version is nicely reduced but how about keeping it simple and verbose?